src/plugins/intel_gna/src/gna2_model_export_helper.cpp

```diff
@@ -278,3 +215,40 @@ void ExportTlvModel(uint32_t modelId,
         THROW_GNA_EXCEPTION << "Not succesfull status returned: " << tlv_status << ", from Gna2ExportTlv() function\n";
     }
 }
+
+void* ExportSueLegacyUsingGnaApi2(uint32_t modelId, uint32_t deviceIndex, Gna2ModelSueCreekHeader* modelHeader) {
+    uint32_t exportConfig;
+    auto status = Gna2ModelExportConfigCreate(gnaUserAllocatorAlignedPage, &exportConfig);
+    GNADeviceHelper::checkGna2Status(status, "Gna2ModelExportConfigCreate");
+
+    status = Gna2ModelExportConfigSetSource(exportConfig, deviceIndex, modelId);
+    GNADeviceHelper::checkGna2Status(status, "Gna2ModelExportConfigSetSource");
+    status =
+        Gna2ModelExportConfigSetTarget(exportConfig, static_cast<Gna2DeviceVersion>(DeviceVersion::GNAEmbedded1_0));
```

Comment:
I would use specific value of Gna2DeviceVersion here due the fact it is hardocded eventually I would use conversion method.

Reply:
Conversion method will be used, omitted by mistake..